shell/platform/android/io/flutter/app/FlutterActivity.java

```diff
@@ -151,4 +153,28 @@ public boolean loadIntent(Intent intent) {
     public FlutterView getFlutterView() {
       return flutterView;
     }
+
+    /**
+     * Returns a named channel for communicating with the Flutter application.
+     *
+     * @param name The logical name String of the channel.
+     * @return a {@link FlutterChannel} using the {@link MessageCodec#STANDARD} message codec.
+     *
+     * @see MessageCodec#STANDARD for the message values supported.
+     */
+    public FlutterChannel createFlutterChannel(String name) {
+        return createFlutterChannel(name, MessageCodec.STANDARD);
+    }
```

Comment:
Why have these functions?  We'd like to keep FlutterActivity as small as possible because people will otherwise need to replicate this code if they make their own activities.